[2024-01-18T22:11:51.861+0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Google_X_mention_score.process_data scheduled__2024-01-01T04:30:00+00:00 [queued]>
[2024-01-18T22:11:51.880+0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Google_X_mention_score.process_data scheduled__2024-01-01T04:30:00+00:00 [queued]>
[2024-01-18T22:11:51.885+0300] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2024-01-18T22:11:51.887+0300] {taskinstance.py:1289} INFO - Starting attempt 1 of 2
[2024-01-18T22:11:51.889+0300] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2024-01-18T22:11:51.951+0300] {taskinstance.py:1309} INFO - Executing <Task(BashOperator): process_data> on 2024-01-01 04:30:00+00:00
[2024-01-18T22:11:51.964+0300] {standard_task_runner.py:55} INFO - Started process 6190 to run task
[2024-01-18T22:11:51.990+0300] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'Google_X_mention_score', 'process_data', 'scheduled__2024-01-01T04:30:00+00:00', '--job-id', '52', '--raw', '--subdir', 'DAGS_FOLDER/google_mention_score.py', '--cfg-path', '/tmp/tmpr3e5c4c5']
[2024-01-18T22:11:51.997+0300] {standard_task_runner.py:83} INFO - Job 52: Subtask process_data
[2024-01-18T22:11:52.147+0300] {task_command.py:389} INFO - Running <TaskInstance: Google_X_mention_score.process_data scheduled__2024-01-01T04:30:00+00:00 [running]> on host xflow
[2024-01-18T22:11:52.270+0300] {taskinstance.py:1516} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=Google_X_mention_score
AIRFLOW_CTX_TASK_ID=process_data
AIRFLOW_CTX_EXECUTION_DATE=2024-01-01T04:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-01-01T04:30:00+00:00
[2024-01-18T22:11:52.274+0300] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-01-18T22:11:52.275+0300] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /home/angry/mlops_4/scripts/process_data.py']
[2024-01-18T22:11:52.314+0300] {subprocess.py:86} INFO - Output:
[2024-01-18T22:11:54.065+0300] {subprocess.py:97} INFO - Command exited with return code 0
[2024-01-18T22:11:54.122+0300] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=Google_X_mention_score, task_id=process_data, execution_date=20240101T043000, start_date=20240118T191151, end_date=20240118T191154
[2024-01-18T22:11:54.198+0300] {local_task_job.py:212} INFO - Task exited with return code 0
[2024-01-18T22:11:54.262+0300] {taskinstance.py:2596} INFO - 1 downstream tasks scheduled from follow-on schedule check
